package.json

```diff
@@ -166,13 +167,6 @@
   "optionalDependencies": {
     "fsevents": "^2.1.2"
   },
-  "resolutions": {
```

Comment:
@BYK I've removed these since they are no longer needed (I believe most of them were from very old version of storybook) - in the future I may want to split off storybook from this main package.json.